server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -1308,6 +1316,13 @@ assert getLocalNode().equals(clusterState.getNodes().get(getLocalNode().getId())
                 leaderChecker.setCurrentNodes(publishNodes);
                 followersChecker.setCurrentNodes(publishNodes);
                 lagDetector.setTrackedNodes(publishNodes);
+
+                PersistedState remotePersistedState = remotePersistedStateSupplier.get();
+                if (remotePersistedState == null) {
+                    logger.error("remote persisted state is null");
+                } else {
+                    remotePersistedState.setLastAcceptedState(clusterState);
```

Comment:
at this stage, the new cluster state is not persisted locally on the active cluster manager node